scripts/minikube-minishift-setup-env.sh

```diff
@@ -7,10 +7,11 @@ shout() {
 }
 
 # Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
-mkdir bin artifacts
+mkdir -p go/bin artifacts
 # Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
 # for eg go get ginkgo later on will produce ginkgo binary in GOBIN
-export GOBIN="`pwd`/bin"
+export GOPATH="`pwd`/go"
```

Comment:
similarly to before, this depends on the current directory, we should try to avoid that

Reply:
Uniqie dir will be ensured by ci-firewall with https://github.com/mohammedzee1000/ci-firewall/pull/24